Examples/Swift/WebAPIDataExample.swift

```diff
@@ -86,9 +86,13 @@ class WebAPIDataExample_Swift: UIViewController, MGLMapViewDelegate {
 
             // Try matching the exact point first.
             let point = sender.location(in: sender.view!)
-            for f in mapView.visibleFeatures(at: point, styleLayerIdentifiers:layerIdentifiers)
-              where f is MGLPointFeature {
-                showCallout(feature: f as! MGLPointFeature)
+            for feature in mapView.visibleFeatures(at: point, styleLayerIdentifiers: layerIdentifiers)
+              where feature is MGLPointFeature {
+                guard let selectedFeature = feature as? MGLPointFeature else {
+                    self.displayWarning(description: "Failed to cast selected feature as MGLPointFeature")
```

Comment:
This example has gained a lot of error-handling and UI code — what was the rationale for this approach (versus `fatalError` and friends)?

Reply:
Taking a second look at this, it seems like this isn't ideal. User-facing alerts probably shouldn't display any mention of internal APIs, so I'll replace these with the other approaches I took with the other examples.